### PR TITLE
Jetpack support with multiple modules

### LINKED ISSUE #29: Support jetpack compose
Add a convenient wrapper around `.resolve(context)`

```
@Composable
fun textRef(textRef: TextRef): String = textRef.resolve(LocalContext.current)

@Composable
fun textRefOrNull(textRef: TextRef?): String? = textRef?.resolve(LocalContext.current)
```

Stuff to think about/find out
- if compose added, how behaves dependents that doesn't use compose
- do we need another artifact for that? (textref-compose)?
- ...

### PR DESCRIPTION
**Pull request by StefMa**
Fixes #29

### CODE REVIEW

File: compose/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        withJavadocJar()
+    }
+}
+
+val projectVersion = version as String
+publishing {
+    publications {
+        register<MavenPublication>("release") {
+            groupId = "com.github.ioki-mobility"
+            artifactId = "TextRef-Compose"
```
Comment (gustavkarlsson) [inline compose/build.gradle.kts:44]: Are these the right ID:s? They don't match up with the instructions in the readme 🤔

**Review by gustavkarlsson — COMMENTED** [review]
I compared this with my own version, and it's almost exactly the same :D

The one major thing that I think should be added is some kind of check for the compose module. I know there are no tests for it, but I there should probably be a job that at least compiles it. What do you think?

File: compose/build.gradle.kts (hunk 1 above)
Comment (StefMa) [reply compose/build.gradle.kts:44]: What does "right" mean? 🤔 😁 
Jitpack works differently and will ignore the `groupId` and `artifactId` defined here anyways.
They are only used for native publishing...

Nevertheless, I think you are right and I should adjust it 👍

**Comment by StefMa** [comment]
Thanks for the feedback @gustavkarlsson 🙏 
I [adjusted the `groupId` and `artifactId`](https://github.com/ioki-mobility/TextRef/pull/37/commits/ebcf58d4d851acbe01a8305d68d5b47df283e769).
I also [added a CI step](https://github.com/ioki-mobility/TextRef/pull/37/commits/011ae99c3e119118e557d51739c36ca76a901256). Good idea!

Please note that jitpack still builds a `com.github.ioki-mobility:TextRef:$version`, but artifacts includes now both artifacts (`core` & `compose`).
So in case you don't want to use `compose` you have to adjust the maven artifact to `com.github.ioki-mobility.TextRef:core:$version` in the next release.

**Comment by gustav-es** [comment]
Yep. I'm aware :)
Thanks for the warning though!

**Pull request merged**